### DISCUSSION TITLE
Restrictions on the Prediction

### DISCUSSION THREAD

**Discussion by ramber1836** [PaddlePaddle/PaddleSpatial#91] [BaiduKDDCup2022] [answered]
Dear all, 

The prediction yielded by your submitted model should not have abnormal values, which means empty prediction, NaN prediction, or the prediction with identical values will be punished w.r.t. the final score your model can obtain. 
Please refer to the newly updated evaluation script.

**Comment by zhoujingbo** [comment, marked as answer]
The online tests have been updated to avoid the false high scores, please refer to the GitHub(https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation) updates for more details.
